xml_inscriptions/bib_only/MA.Bos.MFA.G.84.47.xml

```diff
@@ -66,17 +133,71 @@
       <revisionDesc>
          <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-07-17" who="Jean Wanlass">added metadata, transcription,
+            translation</change>
       </revisionDesc>
    </teiHeader>
-   <!--facsimile element goes here-->
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc/>
+         <graphic url="MA.Bos.MFA.G.84.47.jpg"/>
+      </surface>
+   </facsimile>
+   
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+            <ab> <supplied reason="lost"><gap extent="unknown" unit="character" reason="lost"></gap>ἐπῃνῆσθαι δὲ καὶ τὸν δεῖνα καὶ συγκεχωρῆσθαι αὐτῶι πολ</supplied><unclear>ι</unclear>τεί<unclear>α</unclear><supplied reason="lost">ν</supplied> <lb/>
```

Comment:
You're not supplying the gap, right? The gap is there and you then you're positing a supplement, so you have the logic a little backwards here. Also where is the supplement coming from? If this is from the parallel that Robert identified you might be better served marking it as `<supplied reason="lost" evidence="parallel">`